ui/src/ensembler/list/ListEnsemblersTable.js

```diff
@@ -46,6 +52,11 @@ export const ListEnsemblersTable = ({
 
   const onTableChange = ({ page = {} }) => onPaginationChange(page);
 
+  const handleDeleteEnsembler = (ensembler) => {
```

Comment:
Nit: Can we call this function `onDeleteEnsembler`? That's the convention we typically use.